storage-operator/pkg/salt/client.go

```diff
@@ -41,38 +49,140 @@ func NewClient(creds *Credential) *Client {
 	}
 }
 
+func (self *Client) PrepareVolume(nodeName string) (string, error) {
```

Comment:
Let's add cancellation using `context` from the start.

Reply:
Can you give more details about that/what we want :slightly_smiling_face: ?

Reply:
Seen with you, all network queries are now using a root context from the reconciliation loop.